Tests/Functional/Readme.md

```diff
@@ -0,0 +1,3 @@
+# Functional tests
+
+In this folder are tests related to the TestBundle. 
```

Comment:
I'm confused by this statement 'tests related to the TestBundle'? Could this be clarified?